force-app/main/default/triggers/ESA_tgr_ExternalCommitteeTrigger.trigger

```diff
@@ -0,0 +1,11 @@
+trigger ESA_tgr_ExternalCommitteeTrigger on External_Committee__c (before insert,after insert) {
```

Comment:
@deepakmulamalla Similar to the trigger handler class, the trigger should also have a purpose on why the trigger event is run in Before and AfterInsert. Please update.